packages/next-playwright/README.md

```diff
@@ -0,0 +1,37 @@
+# @next/playwright
+
+### Install `@next/playwright` in your project
+
+```sh
+npm install -D @next/playwright
+```
+
+### Update `playwright.config.ts`
+
+```javascript
+import { defineConfig } from '@playwright/test'
+
+export default defineConfig({
+  webServer: {
+    command: 'pnpm dev -- --experimental-test',
+    url: 'http://localhost:3000',
+  },
+})
+```
+
+### Use the `@next/playwright` to create tests
+
+```javascript
+import { expect } from '@playwright/test'
+import { test } from '@next/playwright'
```

Comment:
I wonder if `@next/playwright` should just re-export everything from Playwright so that it simplifies which `test` import to use?

Reply:
Yeah. Could be a good idea. I thought about this as well.

Reply:
Agreed, we do this internally from our custom fixtures too.